debugger/src/main/kotlin/org/rust/debugger/runconfig/RsLocalDebugProcess.kt

```diff
@@ -33,6 +35,24 @@ class RsLocalDebugProcess(
         }
     }
 
+    fun loadRustcSources(sysroot: String, exePath: String) {
+        postCommand { driver ->
+            // TODO: find more convenient way to get the SHA-1
```

Comment:
Can we parse commit hash in `scrapeRustcVersion` and add it to the `RustcVersion`?